dask/array/core.py

```diff
@@ -1151,6 +1151,12 @@ def imag(self):
     def conj(self):
         return conj(self)
 
+    def view(self, dtype):
+        mult = np.empty((1,), dtype=self.dtype).view(dtype=dtype).shape[0]
+        out = elemwise(lambda x, dt: x.copy().view(dt), self, dtype)
```

Comment:
Do we really need to do a copy here? The overhead is probably negligible but I'm not sure what we gain here...